libs/partners/google-vertexai/pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[tool.poetry]
+name = "langchain-google-vertexai"
+version = "0.0.1"
+description = "An integration package connecting GoogleVertexAI and LangChain"
+authors = []
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain-core = ">=0.0.12"
+google-cloud-aiplatform = "1.38.1"
```

Comment:
is this different than the `vertexai` imports? do we want to make that one required as well?

Reply:
`vertexai` is part of `google-cloud-aiplatform`